docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,86 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[known-issues-8.0.0]]
+==== Known issues
+*Case migration errors might be logged when upgrading*
+
+You might find the `Failed to migrate user action alerts` error message in your {kib} migration logs when upgrading to {stack} version 8.0.0. This error is incorrectly logged when migrating cases and can be ignored ({pull}124950[#124950]).
+
+Here is an example of an error message you might encounter:
+
+[code block]
+----
+[2022-02-07T20:25:58.614+00:00][ERROR][savedobjects-service] Failed to migrate user action alerts with doc id: 7420fe08-c2ed-51d2-b077-46deb4bf76c9 version: 8.0.0 error: Unexpected token in JSON at position 0
+----
+
+*Existing or new rules that use the legacy alerts index may temporarily fail after upgrading*
+
+After you upgrade to {stack} version 8.0.0, existing and new rules might fail to execute  if their source index is configured to use a legacy alert index pattern created in {stack} version 7.x (`.siem-signals-<space-id>`). Rule failures will likely cause coverage gaps, which will be proportional in time to the scheduled interval of the rule. Rules will start to successfully execute after legacy alerts are no longer within the scheduled time period queried by the rule. Despite this automatic correction, coverage gaps (up to about a minute interval) might still remain ({pull}124327[#124327]).
```

Comment:
Suggestions in **_bold italic:_** 


After you upgrade to {stack} version 8.0.0, existing and new rules might fail to execute  if their source index is configured to use a legacy alert index pattern created in {stack} version 7.x (`.siem-signals-<space-id>`). Rule failures will likely cause **_detection_** coverage gaps, which will be proportional in time to the scheduled interval of the rule. Rules will start to successfully execute after legacy alerts are no longer within the scheduled time period queried by the rule. Despite this automatic correction, coverage gaps **_~~(up to about a minute interval)~~_** might still remain ({pull}124327[#124327]).